File: VB/GrowthWare.WebApplication/Content/Skins/Default/Default.Master.vb
Imports GrowthWare.Framework.Model.Profiles
Imports GrowthWare.WebSupport.Utilities
Imports GrowthWare.WebSupport

Public Class _Default2
    Inherits System.Web.UI.MasterPage

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load
        Dim mSecurityEntityID As Integer = Integer.Parse(ClientChoicesUtility.GetClientChoicesState(AccountUtility.CurrentProfile.Account).Item(MClientChoices.SecurityEntityId).ToString())
        Dim mSecurityEntityProfile As MSecurityEntityProfile = SecurityEntityUtility.GetProfile(mSecurityEntityID)
        formStyles.Attributes("href") = GWWebHelper.RootSite + "/Content/SiteStyles/" + mSecurityEntityProfile.Style + ".css"
    End Sub

End Class

File: CS/GrowthWare.WebApplication/Content/Skins/Default/Default.Master.cs
using GrowthWare.Framework.Model.Profiles;
using GrowthWare.WebSupport;
using GrowthWare.WebSupport.Utilities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace GrowthWare.WebApplication.Content.Skins.Default
{
    public partial class Default : System.Web.UI.MasterPage
    {
        protected void Page_Load(object sender, EventArgs e)
        {
            int mSecurityEntityID = int.Parse(ClientChoicesUtility.GetClientChoicesState(AccountUtility.CurrentProfile().Account)[MClientChoices.SecurityEntityId].ToString());
            MSecurityEntityProfile mSecurityEntityProfile = SecurityEntityUtility.GetProfile(mSecurityEntityID);
            formStyles.Attributes["href"] = GWWebHelper.RootSite + "/Content/FormStyles/" + mSecurityEntityProfile.Style + ".css";

        }
    }
}